shared/chat/inbox/index.native.js

```diff
@@ -96,7 +97,7 @@ class Inbox extends PureComponent<void, Props, {rows: Array<any>}> {
           keyExtractor={this._keyExtractor}
           renderItem={this._renderItem}
           onViewableItemsChanged={this._onViewChanged}
-          getItemLayout={(data, index) => ({length: 64, offset: 64 * index, index})}
+          getItemLayout={(data, index) => ({index, length: 64, offset: 64 * index})}
```

Comment:
this is actually a warning in eslint. we've not moved forward with making it an error. i'm kinda wishing it was something that could be auto fixed by lint. i tend to sort it as i think it makes things cleaner but its def. not required just fyi